### ISSUE TITLE
Keep matched instance in the middle of the screen on jump

### ISSUE THREAD

**Issue by henryoliver** [kevinhwang91/nvim-hlslens#2]
First of all, thank you for creating this useful plugin and sharing it with us. :)

Sometimes the scrolling is annoying depending on how many found instances you have. It would be nice to have a way to "lock" the search (offset) in the middle of the screen as you jump through them.

**Comment by kevinhwang91** [comment]
Do you mean `zz`?

**Comment by henryoliver** [comment]
Yes, exactly, but it would be an option to keep the jumps locked in the middle of the screen.

**Comment by henryoliver** [comment]
Thanks @kevinhwang91! Appreciated your time on that. I'll play around with what you sent, I think it will accomplish what I'm looking for. Thanks again!

**Comment by yochem** [comment]
For me using `vim.api.nvim_set_keymap('n', 'n', 'nzz', { noremap = true })` or `nnoremap n nzz` just works fine without any configuration to the plugin.
